cpp/src/ice2slice/Main.cpp

```diff
@@ -222,6 +222,7 @@ compile(const vector<string>& argv)
                 }
             }
 
+            status |= p->getStatus();
```

Comment:
After we've finished parsing, validation, and code-gen, we want to check if any errors were reported.
We use `|=` because we don't want to overwrite an `EXIT_FAILURE` with an `EXIT_SUCCESS`.

We also don't want to return because we're looping over a list of files to compile. One of them failing shouldn't stop compilation of all the others.